Description: Ignore source only annotations not available in Debian
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/bundles/org.eclipse.jface/src/org/eclipse/jface/resource/LazyResourceManager.java
+++ b/bundles/org.eclipse.jface/src/org/eclipse/jface/resource/LazyResourceManager.java
@@ -17,14 +17,12 @@
 import java.util.LinkedHashMap;
 import java.util.Map;
 
-import org.eclipse.pde.api.tools.annotations.NoReference;
 import org.eclipse.swt.graphics.Device;
 import org.eclipse.swt.graphics.Image;
 
 /**
  * A LRU based ResourceManager Wrapper. Not to be used by clients.
  */
-@NoReference
 public class LazyResourceManager extends ResourceManager {
 	/**
 	 * This LRU Map only holds the DeviceResourceDescriptors which are not
